pyxem/signals/diffraction2d.py

```diff
@@ -1857,12 +1857,9 @@ def get_azimuthal_integral1d(
         mask=None,
         radial_range=None,
         azimuth_range=None,
-        wavelength=None,
         inplace=False,
         method="splitpixel",
         sum=False,
-        lazy_result=None,
```

Comment:
If `lazy_result` is removed here, should `lazy_output` be added?

Reply:
I was just going to let `**kwargs` handle that. That is how the azimuthalIntegral2D works. We could add it to the doc string if we want?